FizzBuzz/src/test/java/rocks/kata/FizzBuzzTest.java

```diff
@@ -2,14 +2,49 @@
 
 import static org.junit.Assert.assertTrue;
 
-import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
-public class FizzBuzzTest 
-{
-    
-    @Test
-    public void test()
-    {
-        
+public class FizzBuzzTest {
+
+    private FizzBuzz fizzPattern = new FizzBuzzPattern();
+    private FizzBuzz fizzStringBuilder = new FizzBuzzStringBuilder();
+    private FizzBuzz fizzLiterals = new FizzBuzzLiterals();
+    private FizzBuzz fizzDivisibilityCheck = new FizzBuzzDivisibilityRules();
```

Comment:
Osobiście polecam w takim przypadku wykorzystanie wzorca metoda szablonowa (template method) w testach, gdzie podobna logika i elementy się duplikują - w szczególności tutaj duplikuje się de facto wszystko w testach, a zmienia się tylko strategia wykonywania, którą wstrzykujemy do testu. Dzięki temu widać, dla której klasy konkretnie, które przypadki nie przechodzą, a jednocześnie nie duplikuje się kodu i zmiany są stosowane do wszystkich klas testowych od razu.

Reply:
Zrefaktorowałam testy, zerknij proszę jak to teraz wygląda. Pozostałe zmiany zaimplementowałam zgodnie ze wskazówkami.